fml/raster_thread_merger.cc

```diff
@@ -91,7 +91,9 @@ bool RasterThreadMerger::IsOnPlatformThread() const {
   return MessageLoop::GetCurrentTaskQueueId() == platform_queue_id_;
 }
 
-bool RasterThreadMerger::IsOnRasterizingThread() const {
+bool RasterThreadMerger::IsOnRasterizingThread() {
+  std::scoped_lock lock(mutex_);
```

Comment:
nits: instead, can update the `IsMergedUnSafe` below to `IsMerged()`

Reply:
I would rather prevent the merging from happening during the entire duration of this call.